### ISSUE TITLE
PVC and bucket size limit exhaustion

### ISSUE THREAD

**Issue by roman-timoshevskii** [yandex-cloud/k8s-csi-s3#59]
Hello,

Could you please explain how the storage size limiting works with this provisioner?
I've tested it by creating storage class in GeeseFS mode, set limit to PVC and bucket itself but I was able to write much more data than allowed by both PVC size and Bucket size.

Looks like PVC size is completely ignored and Bucket size current utilization is being updated with some significant delay.
Could you please explain if it's expected behaviour or it can be configured somehow?

Thanks!

**Comment by vitalif** [comment]
Yes, standard S3 doesn't have quota support in API, so it's in fact ignored. csi-s3 has no way to set it.
I think it may be possible to implement for Yandex S3 using cloud APIs, but this implementation would be limited to Yandex S3.

**Comment by Nils98Ar** [comment]
@vitalif Could the limit be implemented somewhere on the k8s-csi-s3 / geesefs side?
